docker-compose.example.yml

```
@@ -0,0 +1,40 @@
+version: '3'
+
+services:
+  fhir:
+    depends_on:
+      - mongo
+      - redis
+    environment:
+      # Change this for public location of bulk-export-server this should be the FQDN and location of where the
+      # bulk-export container is made public to users. ex. https://abacus.example.com/bulk-export
+      PUBLIC_BULK_SERVER: http://localhost:3000
+      DB_HOST: mongo
+      DB_PORT: 27017
+      DB_NAME: bulk-export-server
+      REDIS_HOST: redis
+      REDIS_PORT: 6379
+      EXPORT_WORKERS: 2
+    ports:
+      - '3000:3000'
+    volumes:
+      - ./src:/usr/src/app/src
+    command: npm start
```

As mentioned in `docker-compose.yml` this isn't need and the `command` should have been specified in the Dockerfile and also not needed in the compose.
```suggestion
    volumes:
      - ./src:/usr/src/app/src
    command: npm start
```